yarn-project/prover-client/src/orchestrator/orchestrator.ts

```diff
@@ -714,14 +643,7 @@ export class ProvingOrchestrator {
           return this.prover.getPublicKernelProof(request, signal);
         }
       },
-      (result, duration) => {
-        this.emitCircuitSimulationStats(
-          this.getPublicKernelCircuitName(request),
-          request.inputs.toBuffer().length,
-          0,
```

Comment:
This is why the benchmark comment says the public circuit's output increased massively, it was 0 before 👀 

```
circuit_output_size_in_bytes (public-kernel-app-logic): 52,231 (+99%)
circuit_output_size_in_bytes (public-kernel-tail): 7,659 (+96%)
circuit_output_size_in_bytes (public-kernel-teardown): 52,231 (+50%)
circuit_output_size_in_bytes (public-kernel-setup): 52,231 (+50%)
```